### PR TITLE
[#9] Fetch posts with pagination

### PR DESCRIPTION
**Pull request by hoangmirs**
Resolves https://github.com/nimblehq/til-web/issues/9

## What happened 👀

- Add pagination helper
- Use it on the index page

## Insight 📝

N/A

## Proof Of Work 📹

Tests should be passed

### CODE REVIEW

File: src/helpers/pagination.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  items: unknown[];
+  pageNumber: number;
+  pageSize: number;
+}
+
+const DEFAULT_PAGE_SIZE = 10;
+
+const paginate = ({ items, pageNumber, pageSize }: PaginateProps) => {
+  const startIndex = (pageNumber - 1) * pageSize;
+  return items.slice(startIndex, startIndex + pageSize);
```
Comment (Lahphim) [inline src/helpers/pagination.ts:11]: Should add a line here?

File: src/helpers/pagination.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+interface PaginateProps {
+  items: unknown[];
+  pageNumber: number;
+  pageSize: number;
+}
+
+const DEFAULT_PAGE_SIZE = 10;
```
Comment (Lahphim) [inline src/helpers/pagination.ts:7]: Should we move this as a constant file separately?
So we can use that file as a global configuration.
I'm OK to keep in here also.
What do you think?

Comment (hoangmirs) [reply src/helpers/pagination.ts]: Ok to me. But I'm not sure where should we put them. I put them into `config` in https://github.com/nimblehq/til-web/pull/18/commits/48d02e70933d005b05419a804c13c51635f8f8fd
Is it ok?

File: src/helpers/pagination.ts (hunk 1 above)
Comment (hoangmirs) [reply src/helpers/pagination.ts:12]: Fixed in https://github.com/nimblehq/til-web/pull/18/commits/48d02e70933d005b05419a804c13c51635f8f8fd

**Review by tyrro — APPROVED** [review]

**Review by Lahphim — APPROVED** [review]

**Pull request merged**